RedHatInsights/insights-remediations-frontend
# fix(REMEDY-289): use new endpoint for bulk remediation deleting

# Description

Associated Jira ticket: # [REMEDY-289](https://issues.redhat.com/browse/REMEDY-289)

Adopted new endpoint for removing multiple remediations in a single request

# How to test the PR

1. Visit Remediations table
2. Select multiple remediations
3. Click on actions dropdown in the toolbar
4. Click on Delete playbooks
5. Observe new endpoint v1/remediation of type Delete is used

# Before the change


# After the change


# Dependent work link


# Checklist:
